features/waterOrdering/index.js

```diff
@@ -7,12 +7,20 @@ const { randomArrayItem } = require('../../lib')
 
 const DEFAULT_WATER_ORDER_INTERVAL = 5 * 24 * 60 * 60 // 5 days in seconds
 
-const tooMuchOrdersReplies = lastOrderTime => (
+const generateTooMuchOrdersReplies = lastOrderTime => (
   replies.tooMuchOrdersError.map(reply => (
     reply.replace('{{lastWaterOrderCreatedAt}}', lastOrderTime)
   ))
 )
 
+const addRespectNotes = () => {
+  const { askRespect, respectNoteText } = replies
+
+  // I'm not sure how to properly concatenate note to reply to appear it in newline in Slack
+  // \n in Slack may not work
```

Comment:
So have you tested it in slack?

Reply:
@AleksSenkou yeap, tested, here is a screenshots (sent messages from JS code)
![image](https://user-images.githubusercontent.com/19951609/71667869-abd6ba80-2d88-11ea-8df7-270f344b13d0.png)

Reply:
So why have you left these comments then? 😄

Reply:
@AleksSenkou already done)